### ISSUE TITLE
add WKT string for geometry info into Census API

### ISSUE THREAD

**Issue by stevenlis** [uscensusbureau/api#7]
It's very tedious that you have to get the census data (e.g., ACS) via Census API and then find a shapefile to download through TIGER and merge them. Usually you need to merge them based on their FIPS code, but it sometimes changes from year to year. Let's say the Census decides to add a few more tracts in a county or change their area. In other words, you have to download, for example, ACS 2015 shapefile to match with ACS 2015 data. This is very frustrating and also error-prone as some people don't realize this. 

I think a very simple solution would be using WKT to store the geometry info and adding a field in the JSON file returned so that users can covert it to a geo-file after the request from their end.

https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry

**Comment by loganpowell** [comment]
@StevenLi-DS are you aware of the [CitySDK](https://github.com/uscensusbureau/citysdk)?

**Comment by stevenlis** [comment]
@loganpowell Yeah, but CitySDK is a JS lib right inside a general API right? I really wish there could be a Python pkg for it.

**Comment by loganpowell** [comment]
It is a JS lib, yes. Perhaps you can submit a feature request to that repo for it to be made into an API.

**Comment by stevenlis** [comment]
@loganpowell I really think the easier way is to just add a new variable in this existing API as I mentioned on Slack, which doesn't break your workflow and can be very simply covert to a geo file.

**Comment by loganpowell** [comment]
@StevenLi-DS while this might be true if the ownership of the data was in the hands of the technical staff who manage the API, given that the ownership of the data is not centralized - to add data to the API is a years-long process.

**Comment by walkerke** [comment]
@StevenLi-DS I like your idea - though if implementation may not happen in the near term and you are willing to take a look at R, our tidycensus package handles this whole process you describe internally (e.g. https://walker-data.com/tidycensus/articles/spatial-data.html).  Though - the Census TIGER website is down at the moment so it won't work until it is back up.

**Comment by stevenlis** [comment]
Hi, @walkerke. The problem is that this should not be a language or package specific issue and it should be included in the general census API so that both developers and users can benefit from it. In fact, I'm aware of tidycensus and tigris. Both are really interesting R packages, but I'd like to keep my workflow in Python.

**Comment by loganpowell** [comment]
@StevenLi-DS for Python, have you checked out the features available in these packages:
- https://github.com/cenpy-devs/cenpy
- https://github.com/datamade/census

**Comment by stevenlis** [comment]
@loganpowell I haven't looked at cenpy. So far, I'm ok with just using `requests`. I don't think this should not be passed to a third party developers as they also need the geo info for building a package.
